src/lib/ClipboardUtils.js

```diff
@@ -0,0 +1,55 @@
+class ClipboardUtils { // eslint-disable-line no-unused-vars
+    /**
+     * @param {string} text
+     * @returns {boolean}
+     */
+    static copy(text) {
+        // Simplified and typed version of https://github.com/sindresorhus/copy-text-to-clipboard
+        // Additionally added a fix for correctly restoring selections in input fields.
+        const element = document.createElement('textarea');
+
+        element.value = text;
+
+        // Prevent keyboard from showing on mobile
+        element.setAttribute('readonly', '');
+
+        // @ts-ignore: css property contain not known to current ts version
+        element.style.contain = 'strict';
+        element.style.position = 'absolute';
+        element.style.left = '-9999px';
+        element.style.fontSize = '12pt'; // Prevent zooming on iOS
+
+        // store selection to be restored later
+        const selection = /** @type {Selection} */ (document.getSelection());
+        const originalRange = selection.rangeCount > 0 ? selection.getRangeAt(0) : null;
+
+        const activeInput = document.activeElement
+        && (document.activeElement.nodeName === 'INPUT' || document.activeElement.nodeName === 'TEXTAREA')
+            ? /** @type {HTMLInputElement | HTMLTextAreaElement} */ (document.activeElement)
+            : null;
+
+        document.body.append(element);
+        element.select();
```

Comment:
Is this necessary with the following two lines?

Reply:
I think so, yes.